### ISSUE TITLE
After dovpanda import, pandas read_csv no longer works with URL argument

### ISSUE THREAD

**Issue by coljac** [dovpanda-dev/dovpanda#76]
# Brief Description

After importing dovpanda, pd.read_csv(url, ...) no longer works, generating a file not found error.

# System Information

- Notebook / IPython / Python Console 

Any/all

- Python version (required): 

3.7.3

# Minimally Reproducible Code

```
import pandas as pd
df = pd.read_csv("http://www.stat.ufl.edu/~winner/data/pgalpga2008.dat")
print(df.head(3)) # works
import dovpanda
df = pd.read_csv("http://www.stat.ufl.edu/~winner/data/pgalpga2008.dat") # error
```
# Error Messages

```
<ipython-input-4-91539aa51e2d> in <module>
      1 import pandas as pd
----> 2 df = pd.read_csv("http://www.stat.ufl.edu/~winner/data/pgalpga2008.dat")
      3 print(df.head(3))

~/anaconda3/envs/tf2/lib/python3.7/site-packages/dovpanda/base.py in run(*args, **kwargs)
    157                 for pre in pres:
    158                     if self.similar <= pre.stop_nudge:
--> 159                         pre.replacement(arguments)
    160                 ret = f(*args, **kwargs)
    161                 for post in posts:

~/anaconda3/envs/tf2/lib/python3.7/site-packages/dovpanda/core.py in check_csv_size(arguments)
    108 def check_csv_size(arguments):
    109     filename = arguments.get('filepath_or_buffer')
--> 110     if os.path.getsize(filename) > config.MAX_CSV_SIZE:
    111         ledger.tell('File size is very large and may take time to load. '
    112                     'If you would like to avoid format issues before the complete file loads, '

~/anaconda3/envs/tf2/lib/python3.7/genericpath.py in getsize(filename)
     48 def getsize(filename):
     49     """Return the size of a file, reported by os.stat()."""
---> 50     return os.stat(filename).st_size
     51 
     52 

FileNotFoundError: [Errno 2] No such file or directory: 'http://www.stat.ufl.edu/~winner/data/pgalpga2008.dat'
```

**Comment by DeanLa** [comment]
Thanks. dovpanda tries to hint you about the file size and assumes it is in your file system.
It breaks when loading a csv from the web/
Will be fixed for next release.

**Comment by DeanLa** [comment]
issue fixed for release 0.0.5.
